src/operator/tensor/elemwise_unary_op.h

```diff
@@ -476,6 +476,31 @@ void HardSigmoidBackward(const nnvm::NodeAttrs& attrs,
   });
 }
 
+struct ReshapeLikeParam : public dmlc::Parameter<ReshapeLikeParam> {
+  int lhs_begin, rhs_begin;
+  dmlc::optional<int> lhs_end, rhs_end;
+  DMLC_DECLARE_PARAMETER(ReshapeLikeParam) {
+    DMLC_DECLARE_FIELD(lhs_begin).set_default(0).describe(
+        "Defaults to 0. "
+        "The beginning index along which the lhs dimensions are to be "
+        "reshaped. Supports negative indices.");
+    DMLC_DECLARE_FIELD(lhs_end)
+        .set_default(dmlc::optional<int>())
+        .describe("Defaults to None. The ending index to be used, "
+                  "The ending index along which the lhs dimensions are to be "
```

Comment:
This describe comment seems to have a spurious sentence fragment in it.

Reply:
Fixed.